routes/authors.js

```diff
@@ -1,8 +1,55 @@
 const express = require("express");
 const generateId = require("uuid/v4");
+const authors = require("../data/authors");
+const body = require('body-parser')
+const books = require("../data/books");
+
+
 
 const router = express.Router();
 
 // Define your endpoints here
 
-module.exports = router;
+router.get('/', (req, res, next) => {
+	const sorter = req.query.sort
+	if (sorter) res.json(authors.map(el => el[sorter]).sort())
+	res.json(authors)
+})
+
+router.get('/=', (req, res, next) => {
+	res.json(authors.map(el => el.name))
+})
+
+
+router.get('/:id', (req, res, next) => {
+	res.json(authors.filter(el => el.id === req.params.id))
+})
+
+router.post('/', (req, res, next) => {
+	const { name, website } = req.body
+	const author = {
+		id: generateId,
```

Comment:
`generateId` is a function; you need to call it.

Reply:
:duck: